### ISSUE TITLE
feature fs - agent mkdir

### ISSUE THREAD

**Issue by klingenm** [metalbear-co/mirrord#2221]
Currently, configuring 

```
{
  "feature": {
    "fs": {
      "read_write": "^/mnt/data/.*"
    }
  }
}
```

will only successfully create "ordinary" file entries in the remote filesystem. If the executable running in `mirrord` tries to create a directory the remote filesystem the mkdir call will actually be called locally.

The current behaviour is quite confusing, as xstat calls are forwarded to the agent (checking if directory exists) but actual mkdir call is done locally.

**Comment by danielloader** [comment]
Yeah this has hit me before too, good to see an issue raised.

**Comment by AlbertZheng** [comment]
Yeah this has hit me too, good to see an issue raised. BTW, `rmdir` also has the same confusing behaviour.
please consider to support too: feature fs - agent rmdir

**Comment by aviramha** [comment]
Done in https://github.com/metalbear-co/mirrord/pull/2949
Thank you @facundopoblete
